addons/im_livechat/static/src/embed/common/compact_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Composer" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-mail-Composer-coreMain')]" position="attributes">
+            <attribute name="t-att-class">{'pe-none': this.thread?.composerDisabled}</attribute>
```

Comment:
That’s not enough. You can still access the buttons or textarea using the "tab" key (and probably through other methods). I suggest we pass a `disabled` prop to the composer, which will handle disabling every part of the composer (buttons, textarea, etc.). We should also update the `o-mail-Composer-bg` CSS so that the composer visually looks disabled (right now, it’s impossible to tell, which is an issue).